assets/js/googlesitekit/data/utils.js

```diff
@@ -206,6 +212,51 @@ export const collectName = ( ...args ) => {
 	return names.shift();
 };
 
+/**
+ * An object of common actions most stores will use.
+ *
+ * @since n.e.x.t
+ *
+ * @return {Object} key/value list of common actions most stores will want.
+ */
+export const commonActions = {
+	/**
+	 * Dispatches an action and calls a control to get the current data registry.
+	 *
+	 * Useful for controls and resolvers that wish to dispatch actions/use selectors
+	 * on the current data registry.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {Object} FSA-compatible action.
+	 */
+	getRegistry() {
+		return { type: GET_REGISTRY };
+	},
+	initialize: initializeAction,
```

Comment:
Why is that here? That seems unrelated and I can't see it used anywhere.

Reply:
It's here as a convenience; I assembled all of the "common actions" I expected more stores would desire here. We use it in the reset store, for instance, and I thought it was nicer to provide here than also have to include separately.

Reply:
Actually though; it still requires manually adding the reducer to the store, so I'll remove this change 🙂